packages/cubejs-base-driver/src/BaseDriver.ts

```diff
@@ -299,6 +302,72 @@ export abstract class BaseDriver implements DriverInterface {
     }
   }
 
+  public getSchemas() {
+    const query = `
+      SELECT table_schema as ${this.quoteIdentifier('schema_name')}
+      FROM information_schema.tables
+      WHERE table_schema NOT IN ('information_schema', 'mysql', 'performance_schema', 'sys', 'INFORMATION_SCHEMA')
+      GROUP BY table_schema
+    `;
+    return this.query<QuerySchemasResult>(query);
+  }
+
+  public getTablesForSpecificSchemas(schemas: QuerySchemasResult[]) {
+    const schemasPlaceholders = schemas.map((_, idx) => this.param(idx)).join(', ');
+    const schemaNames = schemas.map(s => s.schema_name);
+
+    const query = `
+      SELECT table_schema as ${this.quoteIdentifier('schema_name')},
+            table_name as ${this.quoteIdentifier('table_name')}
+      FROM information_schema.tables
+      WHERE table_schema IN (${schemasPlaceholders})
+    `;
```

Comment:
move sql texts out to make it possible to override SQL only instead of whole method